### ISSUE TITLE
update Usage/rust to the 3.0 API

### ISSUE THREAD

**Issue by Tipnos** [biscuit-auth/website#61]
Crate version: biscuit-auth=2.2.0
Doc link: https://www.biscuitsec.org/docs/Usage/rust/#create-an-authorizer

The following is resulting in an error:
```
let mut operation: Fact = "operation($op)".try_into()?;
operation.set("op", "read")?;
```

After some investigation I found that replacing "$" by curly brackets works:
```
let mut operation: Fact = "operation({op})".try_into()?;
operation.set("op", "read")?;
```

**Comment by divarvel** [comment]
This part is a bit outdated indeed (especially with version `3`).
The readme on the crate page now provides up-to-date examples <https://crates.io/crates/biscuit-auth/3.0.0-alpha4>

**Comment by divarvel** [comment]
now that 3.0 api is released, we can indeed update the rust examples :+1:

**Comment by divarvel** [comment]
this should be fixed in #88
